metacat-common-server/src/main/java/com/netflix/metacat/common/server/model/ChildInfo.java

```diff
@@ -0,0 +1,16 @@
+package com.netflix.metacat.common.server.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * ChildInfo.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChildInfo {
```

Comment:
Probably a cleaner interface to have a base RelationInfo Class with name and relationType as members and have ChildInfo and parentInfo extends it